packages/connect-common/scripts/check-all-firmware-revisions.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+PARENT_PATH=$( cd "$(dirname "${BASH_SOURCE[0]}")" || exit ; pwd -P )
+
+cd "$PARENT_PATH/../files/firmware" || exit
+
+for dir in */ ; do
```

Comment:
this `cd` + glob is smart, I wanted to use it as well [here](https://github.com/trezor/data/pull/90/files#diff-174612db9a6f23fe0db7575ea6471cddf33039da8341189af284c0084d0ee632R3) but I did not found a way how to filter out some results

Reply:
Well I learned it from ChatGPT :smile: It lists only directories, but in this case we don't need to filter out any dirs, so it's simpler than in your case..